Batteries/Tactic/Init.lean

```diff
@@ -42,8 +42,8 @@ macro (name := byContra) tk:"by_contra" e?:(ppSpace colGt binderIdent)? : tactic
     | none => Unhygienic.run `(_%$tk)
   `(tactic| first
     | guard_target = Not _; intro $e:term
-    | refine Decidable.byContradiction fun $e => ?_
-    | refine Classical.byContradiction fun $e => ?_)
+    | refine @Decidable.byContradiction _ fun $e => ?_
```

Comment:
This is missing an argument I think

Reply:
Ah yes I suppose it is!

Reply:
On zulip there are now some other fixes which don't rely 
```suggestion
    | refine (Decidable.byContradiction fun $e => ?_ :)
```
I think works, while avoiding having to count arguments.

Reply:
I pushed this fix.